frontend/src/__tests__/cypress/cypress/pages/projects.ts

```diff
@@ -326,9 +336,51 @@ class ProjectDetailsSettingsTab extends ProjectDetails {
   }
 }
 
+class ProjectDetailsOverviewTab extends ProjectDetails {
+  visit(project: string) {
+    super.visitSection(project, 'overview');
+  }
+
+  findDeployedModelServingRuntime(name: string) {
+    return cy
+      .findByTestId('section-overview')
+      .get('div')
+      .contains(name)
+      .parents('.odh-type-bordered-card .model-server')
+      .get('dd');
+  }
+
+  findModelServingPlatform(name: string) {
+    return cy.findByTestId(`${name}-platform-card`);
+  }
+}
+
+class KserveTableRow extends TableRow {
```

Comment:
Maybe we named it incorrectly, but we use it to select deployed model records from the models table in the Models tab. See test called [should list the deployed model in Models tab](https://github.com/TomerFi/odh-dashboard/blob/nim-cypress-e2e-tests/frontend/src/__tests__/cypress/cypress/tests/mocked/projects/modelServingNim.cy.ts#L92).

Reply:
@andrewballantyne there's a decent amount of model serving related duplication between the projects page tests and the model serving page tests, since we have some of the same functionality in 3 places (project details overview tab, project details models tab, model serving page). It maybe should be cleaned up at the same time as some of the refactoring @lucferbux and @emilys314 have mentioned doing as part of the KServe Raw Deployment work.